test/recipes/90-test_store.t

```diff
@@ -559,8 +559,8 @@ sub runall {
     return 1;
 }
 
-# According to RFC8089, a relative file: path is invalid.  We still produce
-# them for testing purposes.
+# According to RFC8089, a relative file: path is invalid.
+# We still support them produce them here for testing purposes.
```

Comment:
Mind rephrasing that?

Reply:
Oops, added the missing 'and':
`We still support them and produce them here for testing purposes.`